src/Reflex/Class.hs

```diff
@@ -286,7 +286,8 @@ class ( MonadHold t (PushM t)
   -- | Create an 'Event' that will occur whenever the currently-selected input
   -- 'Event' occurs
   switch :: Behavior t (Event t a) -> Event t a
-  -- | Create an 'Event' that will occur whenever the input event is occurring -- and its occurrence value, another 'Event', is also occurring
+  -- | Create an 'Event' that will occur whenever the input event is occurring -- and its occurrence value, another 'Event', is also occurring.
+  --   You maybe looking for 'switchHold' 'never' instead.
```

Comment:
`s/maybe/may be`

Reply:
I really like these type of haddocks, because they help a new user of a library to get the "lay of the land" - how to solve problems using the library, and develop a sense of "where everything is".

To that end, I suggest adding a little more detail as to why the user might be considering `switchHold never`. Perhaps something like this?

    Note: You may be looking for @'switchHold' 'never'@ instead, which gives you access to the most recent 'Event t a'.

Reply:
Also: I don't remember how haddock handles spaces. You might want to add some newlines to the comment block so that your note starts on a new line. ([Anti-example: `networkHold`](http://hackage.haskell.org/package/reflex-0.6.3/docs/Reflex-Network.html#v:networkHold))

Reply:
I don't feel "most recent" accurately conveys the promptly/non-promptly concerns. I'd just make sure `switchHold` itself is documented properly and stick with the "you probably don't want this" note

Reply:
I agree with your objection. If you can find some better words to capture the flavour, great, otherwise yeah that's fine.